osu.Game/Rulesets/Objects/SliderPath.cs

```diff
@@ -273,14 +273,6 @@ private Vector2 interpolateVertices(int i, double d)
             return p0 + (p1 - p0) * (float)w;
         }
 
-        public bool Equals(SliderPath other)
-        {
-            if (ControlPoints == null && other.ControlPoints != null)
-                return false;
-            if (other.ControlPoints == null && ControlPoints != null)
-                return false;
```

Comment:
Unrelated, but here's the only location I caught together.
`null` converts to empty span.